sources/packages/web/src/services/http/dto/Student.dto.ts

```diff
@@ -34,6 +34,22 @@ export class UpdateStudentAPIInDTO extends AddressDetailsFormAPIDTO {
   identityProvider: string;
 }
 
+export interface UpdateStudentDetailsAPIInDTO {
+  givenNames: string;
+  lastName: string;
+  birthdate: string;
+  email: string;
+  noteDescription: string;
+}
+
+export interface UpdateStudentDetails {
+  givenNames: string;
+  lastName: string;
+  birthdate: string;
+  email: string;
+  noteDescription: string;
+}
```

Comment:
This file would be exclusive to DTOs. 
Besides that, what is the idea of having the `UpdateStudentDetailsAPIInDTO` and `UpdateStudentDetails` with the same properties?